include/chopper/layout/hierarchical_binning.hpp

```diff
@@ -253,7 +253,7 @@ class hierarchical_binning
                     // if we use the union estimate we plug in that value instead of the sum (weight)
                     // union_estimates[j_prime] is the union of {j_prime, ..., j}
                     // the + 1 is necessary because j_prime is decremented directly after weight is updated
-                    return config.estimate_union ? data->union_estimates[j_prime + 1] : weight;
+                    return (!config.disable_estimate_union) ? data->union_estimates[j_prime + 1] : weight;
```

Comment:
```suggestion
                    return config.disable_estimate_union ? weight : data->union_estimates[j_prime + 1];
```

Seems easier to read than negation?